docs/user-guide/quick-start.md

```diff
@@ -60,12 +52,11 @@ automatically.
 To kill the application and delete its containers and public IP address, do:
 
 ```shell
-$ kubectl delete rc my-nginx
-replicationcontrollers/my-nginx
-$ kubectl delete svc my-nginx
-services/my-nginx
+$ kubectl delete deployment/my-nginx service/my-nginx
```

Comment:
Woops, was about to merge before I saw this inline comment. Will wait until this command is formatted to be comma separated.